.envrc

```diff
@@ -0,0 +1 @@
+use nix
```

Comment:
```suggestion
use nix
watch_file flake.nix
watch_file flake.lock
```

Reply:
Because our shell.nix loads the flake we should add watchers. Otherwise direnv won't reload automatically if flake.nix or flake.lock changes which defeats one of the important features of direnv.

If you're curious, see

https://github.com/direnv/direnv/blob/9f22a3ea7e9fc835bb0569f14b35d756d02a4b7c/stdlib.sh#L1257-L1279

we could also use `use flake` but this won't work for users that don't have flakes enabled in their nix install.